routing.go

```diff
@@ -247,3 +250,13 @@ func NewTenantsResourceRoutes(controller *tenantsController.Controller) []*rest.
 		rest.Post(ApiUrlInternal+"/tenants", controller.ProvisionTenantsHandler),
 	}
 }
+
+func NewTenantDeploymentsRoutes(controller *tenantsController.Controller) []*rest.Route {
```

Comment:
There is already router for `tenantsController` no need for yet another one.
Made sense if there was different controller around, but when using one then it's just extra code.